Probesys/bileto
# fix: Don't show contract startAt in error when maxHours is

## Related issue(s)



https://github.com/Probesys/bileto/issues/29

## How to test manually



- create a new contract, set max hours to a negative number (you’ll have to remove the `min` attribute in the HTML via the devtools)
- check that the startAt input is not in error

## Checklist
